components/src/core/components/PopOver/PopOver.vue

```diff
@@ -42,20 +46,42 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    persistent: {
+      type: Boolean,
+      default: false,
+    },
   },
 
   emits: ['update:show'],
   // eslint-disable-next-line
-  setup: function(props: any) {
+  setup: function (props: any) {
```

Comment:
Seems unnecessary to use `any` here, vue should be able to infer the types from declared prop types. remove // eslint-disable-next-line also

Reply:
This was made by the previous contributor.. now I fixed this any way